vulnfeeds/cpp/main.go

```diff
@@ -73,21 +75,65 @@ func InScopeGitRepo(repoURL string) bool {
 	return true
 }
 
+// Clones repos and tries to convert version tags to commits
+func GitVersionToCommit(versions cves.VersionInfo, repos []string) (v cves.VersionInfo, e error) {
+	// versions is a VersionInfo with AffectedVersions and no FixCommits
+	// v is a VersionInfo with FixCommits included
+	v = versions
+	for _, repo := range repos {
+		Logger.Infof("Cloning %s", repo)
+		r, err := git.Clone(memory.NewStorage(), nil, &git.CloneOptions{
+			URL: repo,
+		})
+		if err != nil {
+			Logger.Warnf("Failed clone %s: %v", repo, err)
+			continue
+		}
+		for _, av := range versions.AffectedVersions {
+			if av.Fixed == "" {
+				continue
+			}
+			// Need to come up with a way to decide when to prefix
+			ref, err := r.Tag("v" + av.Fixed)
+			if err != nil {
+				Logger.Warnf("Failed to find a tag for %q in %s: %v", av.Fixed, repo, err)
```

Comment:
I wonder if these need to be surfaced as "notes" for potential human triagers.

Reply:
Yes. At the moment I'm not generating per-record notes at all, but I see the value in doing this, and will implement this in the not too distant future.